<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1288</width>
    <height>852</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="self_configuration">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Self Configuration</string>
    </property>
   </widget>
   <widget class="QPushButton" name="online_analysis">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Online Analysis</string>
    </property>
   </widget>
   <widget class="QPushButton" name="offline_analysis">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Offline Analysis</string>
    </property>
   </widget>
   <widget class="QPushButton" name="statistics">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Statistics</string>
    </property>
   </widget>
   <widget class="QFrame" name="logo_frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>241</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="user_communication_frame">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>660</y>
      <width>931</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QStackedWidget" name="notebook">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>931</width>
      <height>621</height>
     </rect>
    </property>
    <widget class="Config_Widget" name="config"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1288</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Config_Widget</class>
   <extends>QWidget</extends>
   <header>self_config_notebook_widget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
